run.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash --
```

Comment:
Should this go in `scripts/`?

Reply:
Yes, it should.  Is this script one that is worth keeping?